app/main/helpers/search_helpers.py

```diff
@@ -20,6 +22,32 @@ def get_page_from_request(request):
         return None
 
 
+def get_request_url_without_any_filters(request, filters):
+    """
+    This function will returns the url path without any filters.
+    It will still retain the categories, keyword and lots parameters as well as any others included.
+
+    Args:
+        request: Request Object (import from flask) of the current request.
+        filters: list of all filters from digital market place framework
+
+    Returns:
+        URL path in string format.
+
```

Comment:
Nicer to not have a blank newline at the end of the docstring. Again FYI and future reference only; don't bother fixing for this.